python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -140,22 +141,58 @@ def since_last_cron_tick(
         )
 
     @staticmethod
-    def any_deps_match(condition: "SchedulingCondition") -> "AnyDepsCondition":
+    def any_deps_match(
+        condition: "SchedulingCondition",
+        include_selection: Optional[AssetSelection] = None,
+        exclude_selection: Optional[AssetSelection] = None,
+    ) -> "AnyDepsCondition":
         """Returns a SchedulingCondition that is true for an asset partition if at least one partition
```

Comment:
Can we have single selection argument and support include/exclude with composition somehow?